utils/platform.lua

```diff
@@ -0,0 +1,15 @@
+local M = {}
+
+local info = sys.get_sys_info()
+
+M.CURRENT = info.system_name
```

Comment:
I assume this is available immediately at boot and never changes?

Reply:
Yep